Andriy_Povzun/HW5_Andriy_Povzun/Task5.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+with open('text', 'r') as file:
+    string = file.read()
```

Comment:
do not use names like string, it is a built-in name variable. And in some cases it could cause to Errors

Reply:
Done.